docs/migrations/web-twig/MIGRATION-v3.md

```diff
@@ -256,6 +257,53 @@ Use:
 
 See [`Tooltip` documentation][tooltip-readme] for more details and examples.
 
+### Tooltip: Hidden by default
```

Comment:
```suggestion
### Tooltip: Hidden by Default
```

Reply:
or maybe the By should be capitalized too 🤔

Reply:
This deserves Textlint and Markdown lint.